### DISCUSSION TITLE
Welcome to Hacktoberfest 2021 Discussions!

### DISCUSSION THREAD

**Discussion by vinitshahdeo** [vinitshahdeo/Hacktoberfest2021#1] [Announcements]
## 👋 Welcome!

We’re using this thread as **a place to connect beginners with the community**. Remember that this is a community we build together 💪. 

We hope that you:

  * ❓  Ask doubts about your first PR.
  
  * 👍  Share beginner-friendly issues (& repositories).

  * 💬  Engage with other community members.

  * 🙏  Welcome and help others to finish the [Hacktoberfest](vinitshahdeo/Hacktoberfest2021) challenge. 

➡️  To get started, **comment below with an introduction of yourself** and tell us about how can we help you.

**Comment by farhanashrafdev** [comment]
Can this repo be considered as beginner friendly?
https://github.com/HackmaFest/HackmaFest-Website

**Reply by vinitshahdeo** [reply]
@mriceflame Yes. Please raise a PR if you haven't yet.

**Comment by vinitshahdeo** [comment]
### 👋  Hello mentors,

We're planning to have quick sync up tomorrow at `10:30 PM IST`. You'll be receiving an invite soon. Please [submit your details](https://forms.gle/ejsMiFFuHbQbpGNGA) if you haven't yet.

➡️  For any queries, please reach out to me or @thisisamank.

> _**Thank you so much for your interest in making a change!  As mentors, you trade away the hours that you could use to pursue your own career goals and spend them on someone else’s, and I salute you for that!**_  🙌 
~ @vinitshahdeo 

---
#### List of Mentors:

- @iamdevvalecha 
- @booleanhunter 
- @bacloud14 
- @parthlimbad 
- @Narasimha1997 
- @pranjalbareth 
- @Kashish121
- @Rishabh-malhotraa 
- @ridsuteri 
- @omrajsharma 
- @Utkarsh1504 
- @Severus-Matthew 
- @Aditya-Jha2002 
- @aedit 
- @deeqakkk 
- @kanak22
- @niharika2810
- @Ismriti12
- @Aryamanz29
- @simran2104
- @ishikaamittal
- @vijetasaini0211
- @Chanchal1603
- @vanshika-srivastava
- @sonali-singh97
- @divyabhushan
- @prathameshborse
- @gauranshkumar
- @mohsin

**Reply by vinitshahdeo** [reply]
Hello folks, a gentle reminder for the meeting tonight at 10:30 pm. 
> Please [let me know](https://twitter.com/Vinit_Shahdeo) if you haven't received the invite.

**Reply by imabp** [reply]
Awesome just received the invite :)

**Reply by vinitshahdeo** [reply]
**Join me to welcome the first set of mentors! Thank you so much for all your support!** ❤️ 

[![hacktoberfest-2021-mentors](https://user-images.githubusercontent.com/20594326/134575350-3a2171a7-d2ea-4d73-895d-920851ff91f2.png)](https://twitter.com/Vinit_Shahdeo/status/1440342986143895565)

**Reply by ghost** [reply]
Lovely.

**Reply by vinitshahdeo** [reply]
Mentors, please have a look at #9 to collaborate with @Amishakumari544 with her idea.

**Reply by vinitshahdeo** [reply]
Welcome to the team 🎉 

[![hacktoberfest-mentors-poster-2](https://user-images.githubusercontent.com/20594326/134745936-4bcb820c-7974-49d2-99a3-ef4e25d91b12.png)](https://twitter.com/Vinit_Shahdeo/status/1441407736743297026)

**Comment by thisisamank** [comment]
Hi mentors, I have tagged all of you so just in case you have missed filling up the google form, you can now. [here](https://forms.gle/WEj1wPmzgkfJzHem7). This is to ensure you don't miss the google meet link of events. Kindly fill up this form as soon as possible because we will be sending you the invite link of today's (25/09/2021) meeting by afternoon.
@vinitshahdeo
@Kashish121
@ishika1727
@iamdevvalecha
@akku555
@pranjalbareth
@prakharagarwal
@thisisamank
@MighTguY
@Rishabh-malhotraa
@himanshusharma89
@parthlimbad
@Utkarsh1504
@AroraShreshth
@Aditya-Jha2002
@Severus-Matthew
@booleanhunter
@omrajsharma
@Narasimha1997
@aedit
@ridsuteri
@deeqakkk
@kanak22
@mohsin
@gauranshkumar
@prathameshborse
@divyabhushan
@sonali-singh97
@Chanchal1603
@vijetasaini0211
@ishikaamittal
@simran2104
@Aryamanz29
@Ismriti12
@niharika2810
@imabp
@maitryeeNaithani
@PulkitSinghDev
@thepranaygupta
@preeti13456
@rajat2502
@praveenscience
@sarthakjdev
@samriddhiambashta
@AbhigyanSrivastav
@ch8n
@anubhav201241
@Ankita297
@Ankita297
@chandrikadeb7
@verma-kunal

**Comment by thisisamank** [comment]
Hi mentors, I have tagged all of you so just in case you have missed filling up the google form, you can now. [here](https://forms.gle/WEj1wPmzgkfJzHem7). This is to ensure you don't miss the google meet link of events. Kindly fill up this form as soon as possible because we will be sending you the invite link of today's (25/09/2021) meeting by afternoon.

@Hemshree 
@amino19 
@akshatvg
@nitish-awasthi
@Jassi10000
@wimpywarlord
@harshita2216
@heraldofsolace
@saloni-15
@naman5665
@carrycooldude
@Prathamesh-Shanbhag
@sobolevn
@Honey_9
@RsTaK
@chiragagrawal9200
@ghulamyazdani
@vedant-kakde
@uma4037
@garvitchittora
@amishakumari544
@kaiwalyakoparkar
@shivammathur
@nehasangeetajha
@arsheelsheikh
@helper-uttam
@harkiratsm
@smilegupta
@StarTrooper08
@felixfaisal
@unnati914
@RutvikJ77
@AnirudhPanda
@sreekaransrinath
@Zain-Muiz
@muskanrastogi
@ashwinexe
@yashwankhade
@KunalRaghav
@Abhinav-26
@fazer1929

**Reply by vinitshahdeo** [reply]
Hello mentors, I have sent an invite for **Session 2.0** today at `10:00 PM IST`. Please let me know if you haven't received the invite.

> PS: @thisisamank I've just sent out the invite (too excited to meet & greet all 🙈 )

**Comment by AnirudhPanda** [comment]
Filled the form at noon today.
Haven't received the invite yet.

**Reply by vinitshahdeo** [reply]
@AnirudhPanda Did you receive the invite now?

**Comment by thisisamank** [comment]
Hello Anirudh,
Can you drop me a Hi on twitter, I'll send you the link personally.

**Reply by kanak22** [reply]
Hey! Actually I saw these threads only now ... So how can I get the link? 😐

**Reply by aminoxix** [reply]
@kanak22: What kind of link, you're concern about? Here's a [form](https://forms.gle/WEj1wPmzgkfJzHem7) need to fill by mentors.
 & those who had already filled, got an invitation for a meeting via mail. :slightly_smiling_face:

**Reply by kanak22** [reply]
That is what I m talking about.. I just saw these some time before.. that means I didn't filled the form and thus hadn't received the link for the meeting

**Reply by vinitshahdeo** [reply]
@kanak22 I hope you should have received the link by now. Please let me know if you haven't yet.

**Reply by kanak22** [reply]
Yes I recieved 😊

**Comment by AnirudhPanda** [comment]
Yeah thanks 😊

**Comment by fazer1929** [comment]
I haven't received the invite yet.
I did fill the form yesterday

**Comment by vanshika-srivastava** [comment]
Haven't received the invite yet .

**Comment by Amishakumari544** [comment]
can I get the invitation now?

**Comment by epicdeepak** [comment]
https://meet.google.com/nnh-nnni-bco

**Comment by thepranaygupta** [comment]
@fazer1929 @vanshika-srivastava @Amishakumari544 
here's the google meet link: https://meet.google.com/nnh-nnni-bco
in case you missed the session here's the link to Praveen's YouTube Stream: https://youtu.be/BMyVbZp8Gn8

**Comment by vanshika-srivastava** [comment]
Thanks , got it noww

**Comment by mohsin** [comment]
Received the invite but was busy with prior engagements. If there was a recording, please share.

**Reply by AbhigyanSrivastav** [reply]
Well there is , you can head here - https://youtu.be/BMyVbZp8Gn8

**Reply by atharvashirdhankar** [reply]
Yes you can have a look over to this live stream

**Comment by AbhigyanSrivastav** [comment]
If Anyone missed Today session you can found a recording on @praveenscience  channel here's the link - https://youtu.be/BMyVbZp8Gn8

**Reply by mohsin** [reply]
Great. Thank you. 😁👍🏽

**Comment by vinitshahdeo** [comment]
Hello mentors,

Hope to see you all on [Twitter Space](https://twitter.com/Vinit_Shahdeo/status/1442000660308770823) tonight at `10:30 PM IST`. 
> Set your reminder [here](https://twitter.com/Vinit_Shahdeo/status/1442000660308770823).

PS: Thank you for joining the session yesterday.

**Comment by lbaweja1999** [comment]
Hi team
First of all Thank you all for this great initiative. I am a beginner at open-source.  I wanted to ask that is the list of this year hacktoberfest repositories out on which we can make our contribution? @vinitshahdeo @iamdevvalecha

**Reply by vinitshahdeo** [reply]
Yes @lbaweja1999, we're compiling a list of beginner-friendly projects [here](https://github.com/vinitshahdeo/Hacktoberfest2021/blob/main/explore/PROJECTS.md).

**Reply by lbaweja1999** [reply]
Okay cool

**Reply by Rishabh-malhotraa** [reply]
@lbaweja1999 If you want to get started, I have some really beginner-friendly issues on the [project ](https://github.com/Rishabh-malhotraa/caucus) I maintain. If you need any more help, we can connect on discord, and I can help you make your first PR.

**Reply by lbaweja1999** [reply]
@Rishabh-malhotraa sure,that would be great.

**Comment by Sanket2055** [comment]
Hello I am Sanket , sophomore at ABV-IIITM , how to get a issue assigned (or is there any need to get a issue assigned  i guess i just need to create a PR) and please share some info regarding the dates when the mentor will be allocated 
Thankyou!

**Reply by vinitshahdeo** [reply]
@Sanket2055 Soon, we'll have an onboarding session for mentees. You can join us today at [Twitter Space](https://twitter.com/Vinit_Shahdeo/status/1442000660308770823) to get all your queries resolved.

**Reply by Sanket2055** [reply]
sir for some reason's i was not able to attend the onboarding session  ,  is the recording available , Thank You!

**Reply by helper-uttam** [reply]
Hey! @Sanket2055 ,
<a href="https://youtu.be/BMyVbZp8Gn8"> Here</a> is the link to watch the recorded session.

**Reply by Sanket2055** [reply]
Thank you @helper-uttam  , but this recording is not of the onboarding session happened yesterday , i.e 26 sep

**Comment by codefromrvk** [comment]
Hey @vinitshahdeo . Just made a PR in inspirational quote repo 4-5 days back. Can you please look on that  and let me know if there is any correction required? 😅🙌🏻

**Reply by vinitshahdeo** [reply]
@codefromrvk You're all done! I will merge all the PRs soon. Thank you.

**Reply by Sanket2055** [reply]
@vinitshahdeo  sir if i do the PR right now , will it be counted in hacktoberfest

**Reply by atharvashirdhankar** [reply]
> sir if i do the PR right now , will it be counted in hacktoberfest
> 

No first we have to register for hacktoberfest with GitHub/Gitlab.
Then it will be counted.
Registration starts 28 Sept🚀

**Reply by Sanket2055** [reply]
okay @StarTrooper08  sir , thanks

**Comment by ashutosh887** [comment]
@vinitshahdeo 
I have been using GitHub for the past six months and I know about most of the workflow in github.
But I am unable to get how may i start contributing to some really good organisations
Could you please give me some suggestions like how may I start with it

**Reply by helper-uttam** [reply]
Try to solve beginner friendly issues at first. Or you can also make changes to their documentation, this will be also counted as a valid contribution to that org. 
Else try to introduce them some new features, and most probably they will guide you through your way😊 because everyone in opensource, loves to help each other.

**Comment by irffanasiff** [comment]
please check the menotrs.md file i submitted my name 3 days ago but still its not updated

**Reply by vinitshahdeo** [reply]
@demonicirfan can you please point me to the PR?

**Comment by vanshika-srivastava** [comment]
@vinitshahdeo If some specific college wants to take up Hacktoberfest as there event , should I share events.md to them ? Also , are there any specific link that I can share - like the digital ocean blogpost page ?

**Reply by vinitshahdeo** [reply]
Hey @vanshika-srivastava 

Please ask them to create an issue [here](https://github.com/vinitshahdeo/Hacktoberfest2021/issues/new?assignees=vinitshahdeo&labels=event-request&template=event-request.md&title=Request+for+an+event). You may share the [official website](https://hacktoberfest.digitalocean.com/). 

`EVENTS.md` is something that we will update once we finalize the speaker.

**Comment by dipenr06** [comment]
The fest will start from 1st Oct so we can't contribute any code at the moment right?

**Reply by atharvashirdhankar** [reply]
Nothing like you can contribute from now itself.
Just the PR/contribution will be counted from/after 1 Oct.

But don't wait for Hacktoberfest to start ,if you have any idea to contribute to open source just do it🚀.

**Comment by vinitshahdeo** [comment]
@vanshika-srivastava are you working on #203?

**Reply by helper-uttam** [reply]
If not, I am happy to work on it :) @vinitshahdeo

**Comment by vanshika-srivastava** [comment]
Jump on Uttam ! Actually I will be sharing a google document where we could
possible / commonly asked questions first , draft them properly along with
the resources and then Add it finally @Vinit Shahdeo
***@***.***> How does this sound?

**Reply by helper-uttam** [reply]
ok let's wait for @vinitshahdeo, then we'll proceed :)

**Comment by vanshika-srivastava** [comment]
There are PR pending , @thisisamank <https://github.com/thisisamank>. @Vinit
Shahdeo ***@***.***> . Anybody merging them ?

**Comment by atharvashirdhankar** [comment]
Hello ,Is the mentee Google form updated??

**Reply by vinitshahdeo** [reply]
Yes, it's updated. Please share.

> https://forms.gle/GJSUTPBJ4Rw4DTMk9

**Comment by vanshika-srivastava** [comment]
Hey all mentors , check the private mentors channel on discord .

@vinitshahdeo @helper-uttam @thisisamank

**Reply by Abhinav-26** [reply]
hey @vanshika-srivastava, I think I am not added in the private mentor channel on discord. Can you please help me with this.

**Reply by vinitshahdeo** [reply]
@PulkitSinghDev can you please add @Abhinav-26 ☝️

**Reply by Abhinav-26** [reply]
Thanks @vinitshahdeo

**Reply by mohsin** [reply]
Me neither. Could you kindly send an invite? Discord ID is moz#9061. Thanks.

**Reply by helper-uttam** [reply]
Mee too🤕

**Reply by vinitshahdeo** [reply]
@PulkitSinghDev ^

**Reply by saloni-15** [reply]
I'm not added as well.

**Reply by chandrikadeb7** [reply]
@PulkitSinghDev I’m not added as well (Chandrika_Deb#5159)

**Reply by atharvashirdhankar** [reply]
Even I'm not added in the mentors channel on discord.
(Atharva Shirdhankar #8971)

**Reply by Chanchal1603** [reply]
Hey @vanshika-srivastava  I'm also not added in mentors private channel

**Reply by AbhigyanSrivastav** [reply]
I'm also not added to the private channel

**Reply by Abhinav-26** [reply]
@vinitshahdeo @PulkitSinghDev Any update on this. and when we gonna have on-boarding call of mentors?

**Comment by preeti13456** [comment]
I have added the pull request for the project. Kindly review and merge it.

________________________________

**Comment by vinitshahdeo** [comment]
Hello all,

Thanks for all your support. 

The official GitHub Community has invited me to speak about our initiative. They will be hosting us [here](https://twitter.com/ALAColombia/status/1442620265310138369).

> Tune in if you're available. It's at PDT timing.

**Reply by atharvashirdhankar** [reply]
Congrats @vinitshahdeo 🎉👏.
Open Source Community FTW.
And Even Kudos to Mentors and Mentees 👏.

**Comment by lbaweja1999** [comment]
When will the registrations be open? Does anyone know?

**Reply by atharvashirdhankar** [reply]
It's 28 Sept but according to IST it will start in Evening, probably 6 - 9 pm.

**Reply by atharvashirdhankar** [reply]
Hope this answers your question 😃

**Reply by lbaweja1999** [reply]
Yeah thanks buddy @StarTrooper08

**Comment by divyabhushan-okta** [comment]
Well done and well-deserved @***@***.***>. I am glad to be a part of this initiative.


Thanks,

Regards,

Divya Bhushan

________________________________

**Comment by atharvashirdhankar** [comment]
Hello everyone , registration for Hacktoberfest 2021 has started now .
You can register with GitHub and Gitlab.

Open Source FTW🚀

**Comment by naman5665** [comment]
Hey 👋, 
Can anyone pls send me the link to the discord channel!!!

**Reply by mohsin** [reply]
Here you go: https://discord.gg/kMFsTNuJ

**Reply by naman5665** [reply]
Thank you @mohsin 🙏😊

**Comment by naman5665** [comment]
Can anyone pls add me to the mentor's private channel??

**Reply by naman5665** [reply]
My discord id:- Naman Budhiraja #4794

**Reply by Jassi10000-zz** [reply]
@naman5665 Am too a mentor , but I don't think there's any private channel
@vinitshahdeo Can you confirm  ?

**Reply by naman5665** [reply]
@Jassi10000 I have seen some messages about the private channel in this discussion channel!!
So, that's why I asked!!!

**Reply by Jassi10000-zz** [reply]
Ohh is that so !!!
I really didn't know

Leme confirm then

**Comment by vinitshahdeo** [comment]
Grateful to each of you - **[Trending 12 on GitHub](https://twitter.com/Vinit_Shahdeo/status/1443194115252883457)**! ❤️  🙏

**Comment by atharvashirdhankar** [comment]
Congrats @vinitshahdeo 👏🥳✨
And kudos to Mentors team, mentees and every participant 🥳🎉

**Comment by Sanket2055** [comment]
hello  , is there any private channel to contact the mentors??

**Reply by verma-kunal** [reply]
you can most definitely join the discord server & interact with the mentors

**Comment by vaarunh** [comment]
When will the new poster be updated, @vinitshahdeo ?

**Reply by vinitshahdeo** [reply]
Hey, do you mean poster for mentors? 

@pranjalbareth can you please look into this?

**Reply by vaarunh** [reply]
Yes for Mentors.

**Reply by atharvashirdhankar** [reply]
Hey even I m not in that mentor poster .

If any help required from my side for making mentors poster I m Happy to help.
Just tag me ( @StarTrooper08 )

**Reply by vaarunh** [reply]
I can also contribute with design. You can share the poster template with both of us and we can work together. @pranjalbareth @vinitshahdeo

**Comment by vinitshahdeo** [comment]
@vanshika-srivastava Please ensure that the speakers push their PPT to `deck/` directory and the event organizers raise PR to add their posters to `assets/poster/events`.

cc/ @thisisamank

**Comment by Ashborn-SM** [comment]
I have doubts regarding the mentors.
Will the mentors be assigned to the contributors individually or something?
How does this all helping others work?
Is there any info about what a mentor should do and how to help?(sorry i couldn't find anything related to this)

Thanks in advance!!

**Comment by TerexitariusStomp** [comment]
Hello I am the developer of HardwoodCrypto. Over the last month and a half our cryptocurrency SaveTrees (on Polygon) has planted over 4,500 trees. We'd like to join in Hacktoberfest to automate tree planting with crypto. Would this be an acceptable repository?  https://github.com/TerexitariusStomp/treeauto

**Comment by vinitshahdeo** [comment]
#### Hello everyone! 👋 

Finally, your wait is over! Join the [onboarding session](https://rb.gy/b6l1mc) on YouTube at 10:30 PM IST Tonight: https://rb.gy/b6l1mc

> **Please set your reminder! Hope to see you there!** 🙏  🙌 

Thanks:)

**Reply by atharvashirdhankar** [reply]
**Great** 🚀🎉

**Comment by SanthoshS20** [comment]
@vinitshahdeo how to identify whether a project is participating in the hacktoberfest 2021? I have seen hacktoberfest label in the project and issues. When i created a PR, it is showing **project not participating** in hacktoberfest website. Can you help me with that?

**Reply by mohsin** [reply]
The best way would be to see the repository's topic list (on the right-side of the repo's main page below about section). For example, check out the same for THIS repository:

<img width="953" alt="Screen Shot 2021-10-02 at 13 03 05" src="https://user-images.githubusercontent.com/1699753/135707893-4f1f3dc8-1084-4dfb-a97c-57230a693053.png">

You can see the **hacktoberfest** tag specified which is what DigitalOcean would count towards Hacktoberfest (not even variations such as hacktoberfest-accepted or hacktoberfest-2021 but just hacktoberfest). Even if a PR or issue has hacktoberfest label but the repo doesn't have the topic, then it would not count towards hacktoberfest. Whereas if a repo has the **hacktoberfest** topic specified, then all it's PRs will be counted (unless the PR violates the general rules--such as being a minor typo fix than a legit PR that has social benefit). I hope that clarifies your doubt.

**Comment by atharvashirdhankar** [comment]
I m still not added as a mentor on discord?

**Comment by vanshika-srivastava** [comment]
Just join the discord , you will get your role Assigned by admin.

**Reply by atharvashirdhankar** [reply]
I have joined the discord (Atharva Shirdhankar #8971)

**Comment by vinitshahdeo** [comment]
Hello 👋 

Hope you're enjoying open-source October! Here, we come again with an opportunity for you to win exciting goodies!  🚨 

#### **You can build a Web/Mobile App or create a CLI tool or GitHub Action and get a chance to win GitHub Stickers!** Your innovative ideas are most welcome. 💡 

- All the details are mentioned [here](https://github.com/vinitshahdeo/hacktoberfest-issue-hunt/discussions/3).
- The best three submissions will receive goodies.
- Post your queries on Discord.
- **The last date for your submission is Friday, October 15.**
- Any submissions after 15th Oct will not be eligible for the goodies.
- **Mentors and Mentees, both can take part in this.**

Let me know if you have any questions.

Thank you!

**Reply by amellptrr** [reply]
hi, i've been created a pull request in your repo, can you merge my change?

**Reply by vinitshahdeo** [reply]
@amellptrr can you please point me to your PR?

**Comment by vanshika-srivastava** [comment]
Will look into it

**Comment by 0xbn0xi0us** [comment]
@vinitshahdeo sir, please can you review my pull request 
Link:-
https://github.com/vinitshahdeo/TwitterSentimentAnalysis/pull/16

**Comment by Jzzopozzo** [comment]
Hi all mentors!
I'm new to contributing in open source and have no idea about how do I get to my first PR.
How will I get personalized mentor assigned?
Could you please help?

**Reply by atharvashirdhankar** [reply]
Which language/tech stack would you like to contribute.
If you are complete beginner I suggest start contribution from documentation.

And yes join the discord channel for help from mentors .

https://discord.gg/qqajmBH6

How you can contribute to open source projects basic steps:
1. Fork the repository
2. Make the asked changes by the maintainer in the issue in the forked repository.
3. Create a Pull Request for the same.
4. And have patience and let the maintainer review your Pull Request.

**Reply by Jzzopozzo** [reply]
Thanks!
I'd like to contribute in python.
I've joined the discord server but again, not much acquainted so don't know how to reach out to mentor

**Reply by atharvashirdhankar** [reply]
You can find mentor on #find-mentor channel of discord.

If you want to contribute to python language repo.
I have one for it

**Reply by atharvashirdhankar** [reply]
https://github.com/StarTrooper08/FlaskCaption

**Reply by atharvashirdhankar** [reply]
Check this out @Jzzopozzo

**Comment by amellptrr** [comment]
Sir, you have'nt merge my Pull Request. Can you check it ? i'm
completely beginner

**Comment by vinitshahdeo** [comment]
Hello everyone 👋 

I have recently created a repo to gather JS resources for interview preparation. If you have encountered any coding problem in JS interview, please [raise a PR](https://github.com/vinitshahdeo/JavaScript-Resources) to add your solution. 

[![Event loop in JavaScript](https://github-readme-stats.vercel.app/api/pin/?username=vinitshahdeo&repo=JavaScript-Resources)](https://github.com/vinitshahdeo/JavaScript-Resources)

**Your contributions are most welcome [here](https://github.com/vinitshahdeo/JavaScript-Resources#contributing).** 🙏

**Comment by vinitshahdeo** [comment]
👋 Hello everyone,

I have opened a few beginner-friendly issues for Hactkoberfest'22 - https://github.com/vinitshahdeo/peerlist-readme-badge
Your contributions are most welcome! 🙏
